doc/nrf/gs_programming.rst

```diff
@@ -119,3 +119,78 @@ After completing the :ref:`manual <build_environment_cli>` or :ref:`automatic <g
+|                                 | Contains the signed version of the application.                                                 | of the multi-core build targets.                                         |
+|                                 | This file is transferred in the real-life update procedure, as opposed to HEX files             |                                                                          |
+|                                 | that are transferred with nrfjprog when emulating an update procedure.                          |                                                                          |
++---------------------------------+-------------------------------------------------------------------------------------------------+--------------------------------------------------------------------------+
+| :file:`net_core_app_update.bin` | Network core update file used to create :file:`dfu_application.zip`.                            | DFU process for the network core of multi-core build targets.            |
+|                                 | This file is transferred in the real-life update procedure, as opposed to HEX files             |                                                                          |
+|                                 | that are transferred with nrfjprog when emulating an update procedure.                          |                                                                          |
++---------------------------------+-------------------------------------------------------------------------------------------------+--------------------------------------------------------------------------+
+| :file:`dfu_application.zip`     | Zip file containing both the MCUboot-compatible update image for one or more cores              | DFU process for both single-core and multi-core applications.            |
+|                                 | and a manifest describing its contents.                                                         |                                                                          |
++---------------------------------+-------------------------------------------------------------------------------------------------+--------------------------------------------------------------------------+
+| :file:`matter.ota`              | :ref:`ug_matter`-specific OTA image that contains a Matter-compliant header                     | DFU over Matter for both single-core and multi-core applications.        |
+|                                 | and a DFU multi-image package that bundles user-selected firmware images.                       |                                                                          |
++---------------------------------+-------------------------------------------------------------------------------------------------+--------------------------------------------------------------------------+
```

Comment:
Add for Zigbee after talking to @sebastiandraus :

:file:`<file_name>.zigbee` | :ref:`ug_zigbee`-specific OTA image that contains the Zigbee application with the Zigbee OTA header used for providing information about the image to the OTA server. The *<file_name>* includes manufacturer's code, image type, file version, optional comment, and the sample name. For example, :file:`127F-0141-01020003-light_switch.zigbee`. | DFU over Zigbee for both single-core and multi-core applications in the |NCS| v2.0.0 and later.